keras_retinanet/bin/evaluate.py

```diff
@@ -89,6 +89,7 @@ def parse_args(args):
     parser.add_argument('--iou-threshold',   help='IoU Threshold to count for a positive detection (defaults to 0.5).', default=0.5, type=float)
     parser.add_argument('--max-detections',  help='Max Detections per image (defaults to 100).', default=100, type=int)
     parser.add_argument('--save-path',       help='Path for saving images with detections.')
+    parser.add_argument('--backbone', help='Backbone model used by retinanet.', default='resnet50', type=str)
```

Comment:
`type=str` is the default, so can be omitted.

Also, please align on the `help` arg :)